packages/adblocker-electron/content.ts

```diff
@@ -64,7 +64,9 @@ function getCosmeticsFilters(data: IBackgroundCallback) {
     },
   );
 
-  ipcRenderer.send('get-cosmetic-filters', window.location.href, id, data);
+  setTimeout(() => {
```

Comment:
It seems that the `ipcRenderer` event isn't registered properly, so I added a `setTimeout` with 1ms and it solved the problem. That's weird.

Reply:
Ha! Interesting, maybe this thing needs a tick to finish (async stuff happening behind the scene?).

Reply:
I have no idea. This didn't happen to me before, although I have been doing basically the same thing.

Reply:
As long as it works, that's great! :D